packages/php-config-printer/src/NodeVisitor/ImportFullyQualifiedNamesNodeVisitor.php

```diff
@@ -28,14 +38,23 @@ public function __construct(
      */
     public function beforeTraverse(array $nodes): ?array
     {
-        $this->nameImports = [];
+        $this->classImports = [];
+        $this->functionImports = [];
+        $this->isFunction = false;
 
         return null;
     }
 
     public function enterNode(Node $node): ?Node
     {
+        if ($node instanceof Node\Expr\FuncCall) {
+            $this->isFunction = true;
+            return null;
+        }
+
         if (! $node instanceof FullyQualified) {
+            $this->isFunction = false;
+
             return null;
         }
```

Comment:
A better way might be using parent node to detect if it's a name in func call.

```php
$parent = $node->getAttribute('parent');
if ($parent instanceof FuncCall) {
    // is func call
	// ...
}
```

Reply:
Thanks. I found out that I needed to add `PhpParser\NodeVisitor\ParentConnectingVisitor` to get this working.